## FASE 4 — Análisis Inferencial (A/B Testing)

**Objetivo:** <br> confirmar estadísticamente los hallazgos detectados 
visualmente en el EDA (Fase 3). Cada caso de esta fase parte de una 
pregunta ya planteada en Fase 3, para dar respaldo estadístico formal 
a lo observado.

**Flujo:** <br>normalidad → homocedasticidad → nº de grupos → test 
correspondiente (t-test / Mann-Whitney / ANOVA+Tukey / Kruskal-Wallis).

In [1]:
import sys
sys.path.append('../')
from src import sp_abtest as sa
import pandas as pd
import scipy.stats as stats

df = pd.read_csv("../data/processed/02_datos_limpios.csv", parse_dates=['fecha_pedido'])
df_pedidos = df

In [5]:
sa.exploracion_df_abtest(df_pedidos,'metodo_pago')

Los principales estadísticos de las columnas categóricas para el grupo PAYPAL son


,count,unique,top,freq
id_cliente,10219,5763,f599a447-f850-4c5e-8dca-e5f63f7a652e,7
product_id,10219,90,P0003,150
metodo_pago,10219,1,paypal,10219
envio_gratis,10219,2,no,7156
devuelto,10219,2,no,9641
canal,10219,4,web,4112
categoria_producto,10219,5,electronica,2109
nombre_producto,10219,90,Auriculares Nova Max,150
marca,10219,9,vantia,2069
proveedor,10219,4,norte logistica,2954


Los principales estadísticos de las columnas numéricas para el grupo PAYPAL son


,count,mean,std,min,25%,50%,75%,max
order_id,10219.0,26012.456209,15042.488644,1.000,12964.000,26052.000,39050.5000,51994.0000
cantidad,10219.0,2.042763,1.207334,1.000,1.000,2.000,3.0000,5.0000
precio_unitario,10219.0,55.950293,46.349299,5.290,28.990,47.130,72.4300,1569.4900
descuento,10219.0,0.070051,0.077123,0.000,0.000,0.050,0.1000,0.3000
importe_total,10219.0,106.521027,121.617666,4.560,39.965,71.940,132.8550,4354.4900
calificacion,10219.0,4.044525,0.961418,1.000,4.000,4.000,5.0000,5.0000
año_ventas,10219.0,2023.500245,1.119532,2022.000,2022.000,2024.000,2025.0000,2025.0000
costo_unitario,10219.0,32.064522,20.449417,5.240,16.120,26.680,42.0300,96.1500
precio_lista,10219.0,56.452239,37.656512,9.700,28.120,45.700,77.9300,211.3600
peso_kg,10219.0,6.249614,3.443284,0.300,2.980,6.890,9.0000,11.9800


Los principales estadísticos de las columnas categóricas para el grupo TARJETA DE CREDITO son


,count,unique,top,freq
id_cliente,17964,7077,61acc4f3-5edf-4aee-9058-62ee1b3c721a,8
product_id,17964,90,P0083,226
metodo_pago,17964,1,tarjeta de credito,17964
envio_gratis,17964,2,no,12619
devuelto,17964,2,no,16855
canal,17964,4,web,7131
categoria_producto,17964,5,electronica,3679
nombre_producto,17964,90,Biografia Kore Pro,226
marca,17964,9,vantia,3735
proveedor,17964,4,norte logistica,5263


Los principales estadísticos de las columnas numéricas para el grupo TARJETA DE CREDITO son


,count,mean,std,min,25%,50%,75%,max
order_id,17964.0,25915.565186,15010.710004,3.000,12930.00000,25879.500,38907.250,52000.0000
cantidad,17964.0,2.041583,1.201048,1.000,1.00000,2.000,3.000,5.0000
precio_unitario,17964.0,56.712047,63.619546,5.450,29.33000,47.070,72.740,4522.2800
descuento,17964.0,0.071020,0.077081,0.000,0.00000,0.050,0.150,0.3000
importe_total,17964.0,107.363376,152.952749,4.220,39.86000,72.145,134.470,11531.8200
calificacion,17964.0,4.043476,0.966253,1.000,4.00000,4.000,5.000,5.0000
año_ventas,17964.0,2023.494545,1.117305,2022.000,2022.00000,2023.000,2024.000,2025.0000
costo_unitario,17964.0,32.531412,20.306699,5.240,17.49000,26.730,43.150,96.1500
precio_lista,17964.0,57.018883,37.061611,9.700,31.07000,45.860,78.750,211.3600
peso_kg,17964.0,6.189986,3.539379,0.300,2.94000,6.890,9.280,11.9800


Los principales estadísticos de las columnas categóricas para el grupo CONTRAREEMBOLSO son


,count,unique,top,freq
id_cliente,5031,3725,5de817b9-2847-441c-89c7-c2e374757ae8,6
product_id,5031,90,P0076,72
metodo_pago,5031,1,contrareembolso,5031
envio_gratis,5031,2,no,3492
devuelto,5031,2,no,4757
canal,5031,4,web,2059
categoria_producto,5031,5,libros,1020
nombre_producto,5031,90,Comic Prisma Basic,72
marca,5031,9,vantia,1098
proveedor,5031,4,norte logistica,1489


Los principales estadísticos de las columnas numéricas para el grupo CONTRAREEMBOLSO son


,count,mean,std,min,25%,50%,75%,max
order_id,5031.0,25890.539455,14983.943540,4.000,12907.0000,25964.000,38693.000,51997.0000
cantidad,5031.0,2.037965,1.198554,1.000,1.0000,2.000,3.000,5.0000
precio_unitario,5031.0,57.934570,68.764145,5.840,29.4100,47.920,74.050,2306.3000
descuento,5031.0,0.071020,0.077292,0.000,0.0000,0.050,0.150,0.3000
importe_total,5031.0,109.962065,163.813204,5.260,39.9250,73.030,133.995,4562.9300
calificacion,5031.0,4.035182,0.975727,1.000,4.0000,4.000,5.000,5.0000
año_ventas,5031.0,2023.491354,1.114461,2022.000,2022.0000,2023.000,2024.000,2025.0000
costo_unitario,5031.0,32.970747,20.912241,5.240,17.4900,26.730,43.150,96.1500
precio_lista,5031.0,58.160753,38.332672,9.700,31.5800,46.140,78.750,211.3600
peso_kg,5031.0,6.308301,3.528955,0.300,2.9800,7.150,9.320,11.9800


Los principales estadísticos de las columnas categóricas para el grupo TARJETA DE DEBITO son


,count,unique,top,freq
id_cliente,12825,6411,feed2f11-2ef0-4473-b141-aba42b5bd5c7,8
product_id,12825,90,P0045,174
metodo_pago,12825,1,tarjeta de debito,12825
envio_gratis,12825,2,no,8953
devuelto,12825,2,no,12068
canal,12825,4,web,5211
categoria_producto,12825,5,libros,2592
nombre_producto,12825,90,Chaqueta Norte&Co Max,174
marca,12825,9,vantia,2713
proveedor,12825,4,norte logistica,3660


Los principales estadísticos de las columnas numéricas para el grupo TARJETA DE DEBITO son


,count,mean,std,min,25%,50%,75%,max
order_id,12825.0,26212.998674,14970.631611,5.000,13284.000,26227.000,39120.000,51998.0000
cantidad,12825.0,2.060975,1.200122,1.000,1.000,2.000,3.000,5.0000
precio_unitario,12825.0,55.758886,49.307104,5.090,29.140,46.840,71.880,2011.6400
descuento,12825.0,0.069996,0.076268,0.000,0.000,0.050,0.100,0.3000
importe_total,12825.0,106.325842,113.317469,4.670,40.310,73.010,133.990,3753.4700
calificacion,12825.0,4.054893,0.966830,1.000,4.000,4.000,5.000,5.0000
año_ventas,12825.0,2023.512593,1.116017,2022.000,2023.000,2024.000,2025.000,2025.0000
costo_unitario,12825.0,32.332861,20.452549,5.240,17.000,26.730,42.030,96.1500
precio_lista,12825.0,56.750058,37.410975,9.700,31.070,45.860,77.930,211.3600
peso_kg,12825.0,6.200391,3.518033,0.300,2.950,6.490,9.280,11.9800


Los principales estadísticos de las columnas categóricas para el grupo TRANSFERENCIA son


,count,unique,top,freq
id_cliente,5181,3834,6afebbfb-bb19-486b-a972-164adeb203ee,5
product_id,5181,90,P0011,74
metodo_pago,5181,1,transferencia,5181
envio_gratis,5181,2,no,3572
devuelto,5181,2,no,4877
canal,5181,4,web,2051
categoria_producto,5181,5,libros,1079
nombre_producto,5181,90,Tablet Prisma Pro,74
marca,5181,9,vantia,1111
proveedor,5181,4,norte logistica,1506


Los principales estadísticos de las columnas numéricas para el grupo TRANSFERENCIA son


,count,mean,std,min,25%,50%,75%,max
order_id,5181.0,25880.944798,15054.740452,23.000,12820.000,25794.000,39277.000,51976.0000
cantidad,5181.0,2.042077,1.188818,1.000,1.000,2.000,3.000,5.0000
precio_unitario,5181.0,57.049396,52.442201,5.180,29.410,47.680,73.320,1625.9300
descuento,5181.0,0.070421,0.076884,0.000,0.000,0.050,0.100,0.3000
importe_total,5181.0,108.516263,144.799066,4.400,40.190,72.450,134.050,5944.6600
calificacion,5181.0,4.035900,0.970543,1.000,4.000,4.000,5.000,5.0000
año_ventas,5181.0,2023.495657,1.126305,2022.000,2022.000,2023.000,2025.000,2025.0000
costo_unitario,5181.0,32.236642,20.457419,5.240,17.000,26.730,42.030,96.1500
precio_lista,5181.0,56.537655,37.265738,9.700,31.070,45.700,72.840,211.3600
peso_kg,5181.0,6.188803,3.538788,0.300,2.950,6.490,9.280,11.9800


Los principales estadísticos de las columnas categóricas para el grupo DESCONOCIDO son


,count,unique,top,freq
id_cliente,780,750,c85b90e2-9315-47d2-a23f-90886aba96dc,2
product_id,780,90,P0042,16
metodo_pago,780,1,desconocido,780
envio_gratis,780,2,no,541
devuelto,780,2,no,743
canal,780,4,web,323
categoria_producto,780,5,electronica,174
nombre_producto,780,90,Pantalon Norte&Co Max,16
marca,780,9,vantia,172
proveedor,780,4,norte logistica,226


Los principales estadísticos de las columnas numéricas para el grupo DESCONOCIDO son


,count,mean,std,min,25%,50%,75%,max
order_id,780.0,25809.371795,15177.819243,84.000,13007.25000,26010.500,38948.2500,51984.0000
cantidad,780.0,2.101282,1.231278,1.000,1.00000,2.000,3.0000,5.0000
precio_unitario,780.0,54.408167,36.172165,6.820,27.32250,46.085,72.2200,241.7900
descuento,780.0,0.065577,0.076410,0.000,0.00000,0.050,0.1000,0.3000
importe_total,780.0,105.833141,102.035191,5.260,40.21750,72.295,130.3825,980.7300
calificacion,780.0,4.024359,0.982868,1.000,4.00000,4.000,5.0000,5.0000
año_ventas,780.0,2023.500000,1.118751,2022.000,2022.75000,2023.500,2024.2500,2025.0000
costo_unitario,780.0,32.557346,20.890110,5.240,17.00000,26.730,43.1500,96.1500
precio_lista,780.0,57.160731,39.586649,9.700,28.12000,45.400,77.9300,211.3600
peso_kg,780.0,6.267692,3.457116,0.300,3.13000,6.890,9.0000,11.9800


### Selección de variables de control y métricas

Las variables de control y métricas de esta fase se seleccionaron tras 
un estudio exploratorio en `00_explorador.ipynb`, usando 
`clasificar_columnas()` (para identificar candidatas por tipo y 
cardinalidad) y `exploracion_df_abtest()` (para revisar en detalle el 
comportamiento por grupo de las candidatas más prometedoras).

### Tablas necesarias (clientes y productos)

In [2]:
clientes = df.groupby('id_cliente').agg(
    gasto_total=('importe_total_wz', 'sum'),
    n_pedidos=('order_id', 'count'),
    ciudad=('ciudad', 'first'),
    segmento_cliente=('segmento_cliente', 'first'),
    ingresos_anuales=('ingresos_anuales', 'first'),
).reset_index()

productos = df.groupby('product_id').agg(
    precio_medio=('precio_unitario', 'mean'),
    costo_medio=('costo_unitario', 'mean'),
    categoria=('categoria_producto', 'first'),
).reset_index()
productos['margen'] = productos['precio_medio'] - productos['costo_medio']

### Caso A — gasto_total ~ segmento_cliente (Cliente, 4 grupos)

**Origen en Fase 3:** en el EDA de Clientes, el gasto medio por 
segmento apenas variaba (671€-689€). Se confirma aquí con test formal.

In [3]:
sa.normalidad(clientes, ['gasto_total'])
sa.homocedasticidad(clientes, 'segmento_cliente', ['gasto_total'])


Para la columna GASTO_TOTAL los datos NO siguen una distribución normal
Para la columna GASTO_TOTAL las varianzas SÍ son homgéneas entre grupos, SI hay HOMOCEDASTICIDAD


In [4]:
sa.decidir_test(clientes, 'segmento_cliente', ['gasto_total'])

--- GASTO_TOTAL | normalidad=False | homocedastico=True | n_grupos=4 ---
H=2.3883, p=0.4958
Para la métrica GASTO_TOTAL, las medianas SI son iguales entre los grupos de segmento_cliente (Kruskal-Wallis no significativo)


In [5]:
sa.kruskal(clientes, 'segmento_cliente', ['gasto_total'])

H=2.3883, p=0.4958
Para la métrica GASTO_TOTAL, las medianas SI son iguales entre los grupos de segmento_cliente (Kruskal-Wallis no significativo)


**Resultado:** Shapiro NO normal, Levene SÍ homocedástico → Kruskal-Wallis. 
H=2,3883; p=0,4958 → **no se rechaza H0**. Confirma lo visto en el EDA: 
segmento_cliente no diferencia el gasto real (medianas: 621-637€).

### Caso B — importe_total_wz ~ canal (Pedido, 4 grupos)

**Origen en Fase 3:** en el EDA de Canales, la mediana de importe era 
casi idéntica entre los 4 canales (71-74€).

In [6]:
sa.normalidad(df_pedidos, ['importe_total_wz'])
sa.homocedasticidad(df_pedidos, 'canal', ['importe_total_wz'])
sa.kruskal(df_pedidos, 'canal', ['importe_total_wz'])

Para la columna IMPORTE_TOTAL_WZ los datos NO siguen una distribución normal
Para la columna IMPORTE_TOTAL_WZ las varianzas SÍ son homgéneas entre grupos, SI hay HOMOCEDASTICIDAD
H=2.9307, p=0.4024
Para la métrica IMPORTE_TOTAL_WZ, las medianas SI son iguales entre los grupos de canal (Kruskal-Wallis no significativo)


In [7]:
sa.decidir_test(df_pedidos, 'canal', ['importe_total_wz'])

--- IMPORTE_TOTAL_WZ | normalidad=False | homocedastico=True | n_grupos=4 ---
H=2.9307, p=0.4024
Para la métrica IMPORTE_TOTAL_WZ, las medianas SI son iguales entre los grupos de canal (Kruskal-Wallis no significativo)


**Resultado:** NO normal → Kruskal-Wallis. H=2,9307; p=0,4024 → 
**no se rechaza H0**. Confirma que el canal no influye en el importe.

### Caso C — importe_total_wz ~ envio_gratis (Pedido, 2 grupos)

**Origen en Fase 3:** contrario a lo esperado, el EDA mostró que 
envío gratis no se asociaba a mayor importe (72,49€ vs 72,60€).

In [8]:
sa.normalidad(df_pedidos, ['importe_total_wz'])
sa.homocedasticidad(df_pedidos, 'envio_gratis', ['importe_total_wz'])
sa.mannwhitneyu(df_pedidos, 'envio_gratis', ['importe_total_wz'])

Para la columna IMPORTE_TOTAL_WZ los datos NO siguen una distribución normal


Para la columna IMPORTE_TOTAL_WZ las varianzas SÍ son homgéneas entre grupos, SI hay HOMOCEDASTICIDAD
U=284698618.0000, p=0.9573
Para la métrica IMPORTE_TOTAL_WZ, las medianas SI son iguales, es decir, NO hay deferencias significativas entre grupos


In [9]:
sa.decidir_test(df_pedidos,'envio_gratis', ['importe_total_wz'])

--- IMPORTE_TOTAL_WZ | normalidad=False | homocedastico=True | n_grupos=2 ---
U=284698618.0000, p=0.9573
Para la métrica IMPORTE_TOTAL_WZ, las medianas SI son iguales, es decir, NO hay deferencias significativas entre grupos


**Resultado:** NO normal → Mann-Whitney. U=284.698.618; p=0,9573 → 
**no se rechaza H0**. Confirma con rigor lo que ya sorprendía en el EDA.

### Caso D — margen ~ categoria_producto

**Origen en Fase 3:** en el EDA de Productos, el margen medio SÍ 
variaba claramente entre categorías (electrónica 14,72€ vs hogar 
29,29€). Este caso se ejecuta a **dos niveles** para mostrar por qué 
la unidad de análisis importa tanto como el test elegido.

#### D.1 — A nivel PRODUCTO (n=90) — NO HACER ASÍ

In [10]:
sa.normalidad(productos, ['margen'])
sa.homocedasticidad(productos, 'categoria', ['margen'])
sa.kruskal(productos, 'categoria', ['margen'])

Para la columna MARGEN los datos NO siguen una distribución normal
Para la columna MARGEN las varianzas SÍ son homgéneas entre grupos, SI hay HOMOCEDASTICIDAD
H=7.4365, p=0.1145
Para la métrica MARGEN, las medianas SI son iguales entre los grupos de categoria (Kruskal-Wallis no significativo)


In [11]:
sa.decidir_test(productos, 'categoria', ['margen'])

--- MARGEN | normalidad=False | homocedastico=True | n_grupos=5 ---
H=7.4365, p=0.1145
Para la métrica MARGEN, las medianas SI son iguales entre los grupos de categoria (Kruskal-Wallis no significativo)


**Resultado D.1:** H=7,4365; p=0,1145 → **NO significativo**. 

⚠️ **Por qué NO se debe concluir aquí:** `margen` es un atributo fijo 
por producto (costo_unitario no varía entre pedidos de un mismo 
producto), así que al construir `productos` (1 fila = 1 producto) solo 
hay **n=90**, repartidos en 5 categorías (~18 productos cada una). Con 
tan poca muestra, aunque la diferencia de medias sea grande en términos 
absolutos, el test no tiene potencia suficiente para declararla 
significativa. Concluir "no hay diferencia" aquí sería un error de 
interpretación, no un hallazgo real.



#### D.2 — A nivel PEDIDO (n=52.000) — versión correcta

In [12]:
df_pedidos['margen_pedido'] = df_pedidos['precio_unitario'] - df_pedidos['costo_unitario']

sa.normalidad(df_pedidos, ['margen_pedido'])
sa.homocedasticidad(df_pedidos, 'categoria_producto', ['margen_pedido'])
sa.kruskal(df_pedidos, 'categoria_producto', ['margen_pedido'])

Para la columna MARGEN_PEDIDO los datos NO siguen una distribución normal
Para la columna MARGEN_PEDIDO las varianzas SÍ son homgéneas entre grupos, SI hay HOMOCEDASTICIDAD
H=1034.1079, p=0.0000
Para la métrica MARGEN_PEDIDO, al menos un grupo de categoria_producto tiene una mediana distinta (Kruskal-Wallis significativo)


In [14]:
sa.decidir_test(df_pedidos, 'categoria_producto', ['margen_pedido'])

--- MARGEN_PEDIDO | normalidad=False | homocedastico=True | n_grupos=5 ---
H=1034.1079, p=0.0000
Para la métrica MARGEN_PEDIDO, al menos un grupo de categoria_producto tiene una mediana distinta (Kruskal-Wallis significativo)


**Resultado D.2:** H=1034,11; p=1,45e-222 → **SIGNIFICATIVO**. 

Repitiendo el mismo margen (constante por producto) a nivel pedido, la 
muestra sube a n=52.000 y el test detecta la diferencia con muchísima 
contundencia. **Es el mismo patrón real en ambos casos — cambia solo 
la potencia estadística disponible para detectarlo.**

**Conclusión del Caso D:** cuando la unidad de análisis natural de una 
variable (producto) tiene poca muestra, conviene replicarla a un nivel 
con más filas (pedido) si esto no distorsiona la pregunta de negocio. 
Aquí es válido porque seguimos preguntando "¿varía el margen entre 
categorías?", no algo distinto por cambiar de nivel.

### Caso E — gasto_total ~ ciudad (Cliente, 10 grupos)

**Origen en Fase 3:** el EDA mostró que el gasto medio por ciudad era 
prácticamente constante (661€-697€).

In [15]:
sa.normalidad(clientes, ['gasto_total'])
sa.homocedasticidad(clientes, 'ciudad', ['gasto_total'])
sa.kruskal(clientes, 'ciudad', ['gasto_total'])

Para la columna GASTO_TOTAL los datos NO siguen una distribución normal
Para la columna GASTO_TOTAL las varianzas SÍ son homgéneas entre grupos, SI hay HOMOCEDASTICIDAD
H=8.0156, p=0.5326
Para la métrica GASTO_TOTAL, las medianas SI son iguales entre los grupos de ciudad (Kruskal-Wallis no significativo)


In [16]:
sa.decidir_test(clientes, 'ciudad', ['gasto_total'])

--- GASTO_TOTAL | normalidad=False | homocedastico=True | n_grupos=10 ---
H=8.0156, p=0.5326
Para la métrica GASTO_TOTAL, las medianas SI son iguales entre los grupos de ciudad (Kruskal-Wallis no significativo)


**Resultado:** H=8,0156; p=0,5326 → **no se rechaza H0**. Confirma que 
ciudad no diferencia el gasto real, tal como viste en Fase 3.

### Anexo: relaciones entre variables (también trazables a Fase 3)

In [18]:
sa.chi_cuadrado_independencia(df_pedidos, 'categoria_producto', 'devuelto')

chi2=2.639, p=0.6199, dof=4
NO hay evidencia de relación entre categoria_producto y devuelto


**Origen:** en Fase 3.4 la tasa de devolución apenas variaba por 
categoría (5,66%-6,06%). Resultado: χ²=2,6393; p=0,6199 → **sin 
relación**, confirmado.

In [19]:
sa.correlacion_regresion(clientes, 'ingresos_anuales', 'gasto_total')

Pearson r=-0.004 (p=0.7137)
Spearman rho=-0.004 (p=0.7423)
                            OLS Regression Results                            
Dep. Variable:            gasto_total   R-squared:                       0.000
Model:                            OLS   Adj. R-squared:                 -0.000
Method:                 Least Squares   F-statistic:                    0.1346
Date:                Thu, 20 Aug 2026   Prob (F-statistic):              0.714
Time:                        23:47:18   Log-Likelihood:                -58381.
No. Observations:                7987   AIC:                         1.168e+05
Df Residuals:                    7985   BIC:                         1.168e+05
Df Model:                           1                                         
Covariance Type:            nonrobust                                         
                       coef    std err          t      P>|t|      [0.025      0.975]
------------------------------------------------------------------

**Origen:** en la matriz de correlación de Fase 3.1, `ingresos_anuales` 
correlacionaba casi 0 con todo. Resultado: r=-0,0041 (p=0,7137), 
R²≈0,000017 → **sin relación lineal útil**, confirmado.

## Conclusiones — Fase 4

### Resumen de los tests realizados

| Caso | Pregunta | Nivel | Origen en Fase 3 | Test | p-valor | Resultado |
|---|---|---|---|---|---|---|
| A | gasto_total ~ segmento_cliente | Cliente | Gasto por segmento casi igual | Kruskal-Wallis | 0,4958 | Sin diferencia |
| B | importe_total_wz ~ canal | Pedido | Ticket medio casi igual por canal | Kruskal-Wallis | 0,4024 | Sin diferencia |
| C | importe_total_wz ~ envio_gratis | Pedido | Importe igual con/sin envío gratis | Mann-Whitney | 0,9573 | Sin diferencia |
| D.1 | margen ~ categoria_producto (producto, n=90) | Producto | Margen SÍ variaba por categoría | Kruskal-Wallis | 0,1145 | Sin diferencia (⚠️ poca potencia) |
| D.2 | margen ~ categoria_producto (pedido, n=52.000) | Pedido | Mismo hallazgo, más potencia | Kruskal-Wallis | 1,45e-222 | **Diferencia significativa** |
| E | gasto_total ~ ciudad | Cliente | Gasto por ciudad casi igual | Kruskal-Wallis | 0,5326 | Sin diferencia |
| Anexo | categoria_producto ~ devuelto | Pedido | Devolución casi igual por categoría | Chi-cuadrado | 0,6199 | Sin relación |
| Anexo | ingresos_anuales ~ gasto_total | Cliente | Correlación casi nula en matriz EDA | Correlación/regresión | 0,7137 | Sin relación |

### Interpretación general

De los 8 tests realizados, **7 confirman con rigor estadístico** lo que 
ya se había detectado visualmente en el EDA de Fase 3: segmento, canal, 
envío gratis, ciudad e ingresos anuales no explican el comportamiento 
de compra ni la devolución en este dataset.

El **Caso D es el único que revela una diferencia real** (margen entre 
categorías), y además ilustra un aprendizaje metodológico clave: el 
mismo patrón real puede parecer "no significativo" o "muy significativo" 
según la unidad de análisis y el tamaño de muestra disponible. Antes de 
concluir la ausencia de un efecto, es necesario comprobar que la muestra 
tiene suficiente potencia estadística para detectarlo.

### Conexión con el EDA (Fase 3)

Cada uno de los 8 casos de esta fase parte directamente de un hallazgo 
visual ya identificado en Fase 3, dando respaldo estadístico formal a 
esas observaciones. El caso del margen (D) refuerza además la 
recomendación de Fase 3 de investigar más a fondo la rentabilidad de 
electrónica frente al resto de categorías.